src/Square3x3.java

```diff
@@ -0,0 +1,206 @@
+/**
+ * Maman 13 - Class Square3x3
+ *
+ * @author Noa Amir
+ * @version 2022a
+ */
+
+public class Square3x3 {
+
+    // initialize final variables
+    private final int SIZE = 3;
+    private final int DEFAULT_CELL_VALUE = -1;
+    private final boolean[] WHICH_NUMBERS_APPEAR = new boolean[SIZE*SIZE+1];
```

Comment:
why is this final?

Reply:
As a final I can use it always and anywhere in the class code, without declaring a local variable each time I want to use it.

In theory I can create a new boolean array in initWhichNumbersAppear method.
something like:
boolean[] whichNumbersAppear;
initWhichNumbersAppear(whichNumbersAppear); 

Than inside initWhichNumbersAppear  method:
whichNumbersAppear = new boolean[SIZE*SIZE+1];
...
...
...

 

What do you think?